prow/logrusutil/logrusutil.go

```diff
@@ -27,39 +27,46 @@ import (
 type DefaultFieldsFormatter struct {
 	WrappedFormatter logrus.Formatter
 	DefaultFields    logrus.Fields
+	PrintLineNumber  bool
 }
 
-// NewDefaultFieldsFormatter returns a DefaultFieldsFormatter,
-// if wrappedFormatter is nil &logrus.JSONFormatter{} will be used instead
-func NewDefaultFieldsFormatter(
-	wrappedFormatter logrus.Formatter, defaultFields logrus.Fields,
-) *DefaultFieldsFormatter {
-	res := &DefaultFieldsFormatter{
-		WrappedFormatter: wrappedFormatter,
-		DefaultFields:    defaultFields,
+// Init set Logrus formatter
+// if DefaultFieldsFormatter.wrappedFormatter is nil &logrus.JSONFormatter{} will be used instead
+func Init(formatter *DefaultFieldsFormatter) {
+	if formatter.WrappedFormatter == nil {
+		formatter.WrappedFormatter = &logrus.JSONFormatter{}
 	}
-	if res.WrappedFormatter == nil {
-		res.WrappedFormatter = &logrus.JSONFormatter{}
-	}
-	return res
+	logrus.SetFormatter(formatter)
+	logrus.SetReportCaller(formatter.PrintLineNumber)
+}
+
+// DefaultInit is a syntax sugar for easier Init
+func DefaultInit(component string) {
```

Comment:
Maybe call this `ComponentInit` or `InitForComponent`?

Reply:
@ibrasho Done.